kratos/python/add_node_to_python.cpp

```diff
@@ -141,6 +141,16 @@ void  AddNodeToPython(pybind11::module& m)
     ;
 
     py::class_<Dof<double>>(m,"Dof")
```

Comment:
At this point I think it makes sense to create an add_dof_to_python.cpp

Reply:
+1 to @rubenzorrilla I was thinking the same. I don't like when the python definition are mixed in an inconsistent way and I expend time to find the file where is defined

Reply:
+1, it is often quite tricky to find the py-exposure things

Reply:
+1

Reply:
done